src/material-experimental/mdc-tabs/harness/tab-group-harness-filters.ts

```diff
@@ -6,4 +6,6 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-export type TabGroupHarnessFilters = {};
+export type TabGroupHarnessFilters = {
```

Comment:
Should this have a doc string?

Reply:
Not sure. The boilerplate and other harnesses don't have a doc string for these either.